docs/intro.rst

```diff
@@ -4,7 +4,8 @@ Intro
 
 ``zyte-parsers`` provides functions that extract specific data from HTML
 elements. The input element can be an instance of either
-:class:`parsel.selector.Selector` or :class:`lxml.html.HtmlElement`.
+:class:`parsel.selector.Selector` or :class:`lxml.html.HtmlElement`. Some
+functions also support passing a string as input.
```

Comment:
I think we should clarify what thi string is: should it be a string with HTML to extractd data from, or should it be text extracted from a HTML element?

Reply:
> extracted from a HTML element

Or elsewhere (e.g. JSON, regexp).